<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Central_AddCars</class>
 <widget class="QDialog" name="Central_AddCars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame
{
background:#111;
}

QPushButton
{
        font-size:15px;
        background:transparent;
        color:#ffffff;
}

QPushButton#HomeButton:hover, QPushButton#AddCars:hover,  QPushButton#EditCars:hover, QPushButton#MyAccount:hover, QPushButton#ViewClients:hover
{
        font-size:15px;
        background:#222;
        color:#ffffff;
        border-radius:50px;
}

QToolButton
{

}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>20</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#5F0000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AddCars">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>    Add Cars</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditCars">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>    Edit Cars</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>26</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(L:/UNI/Projects/Project one/Grind/Project/QT/data/icons/home.png)0 0 0 0 stretch stretch;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MyAccount">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>       My Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewClients">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>       View Clients</string>
    </property>
   </widget>
   <zorder>AddCars</zorder>
   <zorder>EditCars</zorder>
   <zorder>toolButton</zorder>
   <zorder>toolButton_2</zorder>
   <zorder>HomeButton</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>MyAccount</zorder>
   <zorder>ViewClients</zorder>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>821</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#addcarlbl
{
	font-size:24px;
	color:white;
	background-color:#5F0000;
	border-radius:15px;
}

QLabel#WarningLbl
{
	color:red;
	font-size:10px;
}

QLabel#ModelWarning, QLabel#BrandWarning, QLabel#PriceWarning, QLabel#DescriptWarning
{
	color:red;
	font-size:10px;
}

QLabel#CarModel, QLabel#CarBrand,  QLabel#CarPrice, QLabel#CarDescription
{
	color:black;
	font-size:15px;
}

QLineEdit#CarModelLineEdite, QLineEdit#CarBrandLineEdit, QLineEdit#CarPriceLineEdit, QLineEdit#CarDescriptionLineEdit
{
	background-color:#333;
	color:white;
	font-size:14px;
	border-radius:15px;
}

QTextEdit#TextEditDescript
{
	background-color:#333;
	color:white;
	font-size:14px;
}

QPushButton#AddCarButton
{
	font-size:14px;
	background-color:#5F0000;
}
QPushButton#BrowseImages
{
	font-size:10px;
	background-color:#5F0000;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>841</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#5F0000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
        border-image:url(L:/UNI/Projects/Project one/Grind/Project/QT/data/icons/cancel.png);

}

QPushButton:hover
{
        background:#5F0000;
        border-radius:15px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AddCarlbl">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-47</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CarDisplay">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#333;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseImages">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse imgaes</string>
    </property>
   </widget>
   <widget class="QLabel" name="addcarlbl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>      Add a Car</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CarModelLineEdite">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CarModel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Car Model :</string>
    </property>
   </widget>
   <widget class="QLabel" name="CarBrand">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Car Brand :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CarBrandLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CarPrice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Car price :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CarPriceLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CarDescription">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Desciption :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddCarButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Car</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextEditDescript">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="WarningLbl">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ModelWarning">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BrandWarning">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PriceWarning">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DescriptWarning">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
